cypress/integration/materialReceipt/create_material_receipt_with_quality_issue.js

```diff
@@ -78,24 +76,26 @@ describe('Create material receipt with quality issue', function() {
     });
   });
 
-  it('Create Price and ProductCategory', function() {
+  it('Create Price', function() {
     Builder.createBasicPriceEntities(priceSystemName, priceListVersionName, priceListName, false);
+  });
+
+  it('Create Product Category', function() {
     cy.fixture('product/simple_productCategory.json').then(productCategoryJson => {
       Object.assign(new ProductCategory(), productCategoryJson)
         .setName(productCategoryName)
-        .setValue(productCategoryValue)
+        .setValue(productCategoryName)
         .apply();
     });
   });
 
   it('Create product and vendor', function() {
-    Builder.createBasicProductEntities(
-      productCategoryName,
-      productCategoryValue,
+    Builder.createProductWithPriceUsingExistingCategory(
```

Comment:
Why do we need to create the product with price and existing category?

Reply:
Explained on skype: it's needed because we fail when we create the same category each time, inside `Builder.createBasicProductEntities`